README.md

```diff
@@ -13,17 +13,17 @@ Shows how to build a simple example Hugo site.  On the left sidebar there are li
 
 With these installed, you can select the 'Hugo Blog Post' template when you create a new R markdown in R studio.  The R markdown will then knit to a .md file, which you should save to the 'content' directory of the Hugo site.  Hugo handles it from there.
 
-NOTE: `markdowntemplates` by default adds `status: draft` to the markdown header (not rmarkdown file). It is up to you to change that to `type: post` for the post to be published. OR, in Hugo:
-```
-hugo undraft content/post/foo.md
-```
+NOTE: `markdowntemplates` by default adds `status: draft` to the markdown header (not rmarkdown file). It is up to you to change that to `type: post` for the post AFTER they have been reviewed.
```

Comment:
This actually might not be right. I think Hugo is looking for  `draft: True` or `draft: False`